src/metabase/search/postgres/index.clj

```diff
@@ -60,17 +60,18 @@
              [:created_at :timestamp
               [:default [:raw "CURRENT_TIMESTAMP"]]
               :not-null]])
-
           t2/query)
 
+      ;; TODO I strongly suspect that there are more indexes that would help performance, we should examine EXPLAIN.
+
       (t2/query
        (format "CREATE INDEX IF NOT EXISTS %s_tsvector_idx ON %s USING gin (search_vector)"
                (str/replace (str (name active-table) "_" (random-uuid)) #"-" "_")
                (name pending-table))))
```

Comment:
I've been thinking that maybe we should have a migration as a single mustache template so it's easier to understand?

Reply:
Are you a Yesql kinda guy? 😄 

Yeah I see your point, particularly if we add more constraints. On the other hand, in the future I want to avoid nuking this table on deployment if the schema hasn't changed, and that'll be easier to check if the column schema portion is data.

Reply:
nah, not yesql by any stretch, but this piece of code is kind of hard to read (also why random name?)

Reply:
I'll add a comment to this effect in the next PR